.gitignore

```diff
@@ -1,9 +1,11 @@
 cached_departures.json
 config.json
-missing_messages.log
+missing_messages.tmp
+present_messages.tmp
 stop_ids.txt
 audio_command.txt
 speech_command.txt
+github_token.txt
```

Comment:
Question: Why have all these individual config `.txt` files when you also have a `config.json` file that you're parsing? Couldn't these mostly be keys in that file?

Reply:
True. The specification's a little bit set in stone at this point, though, given that this is already deployed on remote machines. The auto-updating doesn't handle gitignored files.

Reply:
Yeah, you're probably right. I can see 3 possible "solutions" to that.

1. Lay hands on all deployments
2. Perform some sort of config file "upgrade" with your code. Obviously, you _can_ do anything with ruby, but I probably wouldn't bother with that complexity
3. Support both, document only the "better" way. Optionally remove support for the "legacy" way once you can do 1 or 2.

All that to say, I guess don't worry about it?